dedupe/manifests/scripts/minikube-start.sh

```diff
@@ -0,0 +1,23 @@
+cd $(dirname $0)
+minikube delete
+minikube start
+eval $(minikube docker-env)
+
+if [ -z $1 ];
+then
+    docker build -t phdi/record-linkage ../../
```

Comment:
@derekadombek does this step assume you have the [phdi repo](https://github.com/CDCgov/phdi) checked out locally and in the same directory as the IDWA repo?

Reply:
The else block did work for me, are you saying that the else block won't work for individuals running docker on apple silicon?

Reply:
ohhh! Thats great to hear! I was unable to get it to work even after figuring out the Rosetta thing yesterday. I even ran different arch minikubes. Did you install the arm64 or the amd64 minikube?

Reply:
amd64, I'm using a 2019 intel macbook pro

Reply:
ah that seems to make sense then. your docker manifest arch inside your minikube instance probably matches up properly to the image arch. I noticed when trying this yesterday that my manifest wouldnt change to match it, even while using the amd64 install. also minikube really loves to complain when I try to use that install haha